homeassistant/components/lg_thinq/entity.py

```diff
@@ -29,35 +33,61 @@ def __init__(
         self,
         coordinator: DeviceDataUpdateCoordinator,
         entity_description: EntityDescription,
-        property: ThinQProperty,
+        idx: str,
```

Comment:
Should we maybe rename it to property_id or something like that? or property, I am not sure what the possible values would be, but `idx` sounds like an index (I mean its the abbreviation) while we don't use it as index but as property identifier

Reply:
Modified to 'property_id'